packages/orama/src/trees/avl.ts

```diff
@@ -53,29 +58,66 @@ function rotateRight<K, V>(node: Node<K, V>): Node<K, V> {
   return left
 }
 
-function findMin<K, V>(node: Node<K, V>): Node<K, V> {
-  return node.left ? findMin(node.left) : node
-}
-
 export function contains<K, V>(node: Node<K, V>, key: K): boolean {
   return !!find(node, key)
 }
 
-export function getSize<K, V>(node: Node<K, V> | null): number {
-  if (!node) {
-    return 0
+export function getSize<K, V>(root: Node<K, V> | null): number {
+  let size = 0;
+  const queue: Array<Node<K, V>> = [];
+
+  if (root !== null) {
```

Comment:
Is there a particular reason why this value set it to null instead of undefined?